.github/workflows/conformance-gateway-api.yaml

```diff
@@ -107,34 +116,46 @@ jobs:
 
       - name: Install Cilium
         run: |
-          helm upgrade -i cilium ./install/kubernetes/cilium \
-            --wait \
-            --namespace kube-system \
-            --set kubeProxyReplacement=strict \
-            --set nodeinit.enabled=true \
-            --set ipam.mode=kubernetes \
-            --set image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci \
-            --set image.tag=${{ steps.vars.outputs.sha }} \
-            --set image.pullPolicy=IfNotPresent \
-            --set image.useDigest=false \
-            --set operator.image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/operator \
-            --set operator.image.suffix=-ci \
-            --set operator.image.tag=${{ steps.vars.outputs.sha }} \
-            --set operator.image.pullPolicy=IfNotPresent \
-            --set operator.image.useDigest=false \
-            --set securityContext.privileged=true \
-            --set gatewayAPI.enabled=true
-
-          kubectl wait -n kube-system --for=condition=Ready -l app.kubernetes.io/part-of=cilium pod --timeout=${{ env.timeout }}
-          kubectl rollout -n kube-system status deploy/coredns --timeout=${{ env.timeout }}
-
-          # To make sure that cilium envoy CRs are available
-          kubectl wait --for condition=Established crd/ciliumenvoyconfigs.cilium.io --timeout=${{ env.timeout }}
-          kubectl wait --for condition=Established crd/ciliumclusterwideenvoyconfigs.cilium.io --timeout=${{ env.timeout }}
+          cilium install ${{ steps.vars.outputs.cilium_install_defaults }}
+
+      - name: Install metallb for LB service
```

Comment:
this would make a great entry to the makefile to develop gateway features for our team!

Reply:
agreed, it's much faster than minikube I can say :+1:.